File: FriendsManager.BBualdo/Data/Models/Friend.cs
using Data.Enums;

namespace Data.Models;

public class Friend
{
  public int Id { get; set; }
  public string? FirstName { get; set; }
  public string? LastName { get; set; }
  public DateOnly LastContactDate { get; set; }
  public ContactTypes LastContactType { get; set; }
  public int DesiredContactFrequency { get; set; }
  public int? CategoryId { get; set; }
  public Category? Category { get; set; }
}

File: FriendsManager.BBualdo/react-ui/src/models/Friend.ts
import { ContactTypes } from "@/enums/ContactTypes";

export interface Friend {
  id: number;
  firstName: string;
  lastName: string;
  lastContactDate: string;
  lastContactType: ContactTypes;
  desiredContactFrequency: number;
  categoryName?: string;
  categoryId?: number;
}
